qmail-qstat.sh

```diff
@@ -1,7 +1,4 @@
-cd QMAIL
-messdirs=`echo queue/mess/* | wc -w`
-messfiles=`find queue/mess/* -print | wc -w`
-tododirs=`echo queue/todo | wc -w`
-todofiles=`find queue/todo -print | wc -w`
-echo messages in queue: `expr $messfiles - $messdirs`
-echo messages in queue but not yet preprocessed: `expr $todofiles - $tododirs`
+messfiles=`find QMAIL/queue/mess/* -type f | wc -l`
+todofiles=`find QMAIL/queue/todo -type f | wc -l`
```

Comment:
The `wc -w` to `wc -l` make sense, find produces one result per line.
It is all based upon the absence of spaces in file names here anyway.

Maybe another case where `-type f` was not widely available.

Reply:
I expect someone that does such special things to the queue to be able to hack this script as well.